packages/components/src/tabs/styles.ts

```diff
@@ -199,8 +199,7 @@ export const Tab = styled( Ariakit.Tab )`
 
 	[aria-orientation='vertical'] & {
 		padding: ${ space( 2 ) } ${ space( 3 ) };
-		padding-left: ${ space( 3 ) };
-		padding-right: ${ space( 3 ) };
+		padding-inline: ${ space( 3 ) };
```

Comment:
Not a big deal and maybe I'm overlooking something but it’s not clear why `padding` shorthand is used first and then `padding-inline`. The latter wouldn’t seem to change anything. If logical properties are preferred then this would be more clear:
```ts
		padding-block: ${ space( 2 ) }
		padding-inline: ${ space( 3 ) }
```
But it seems like in this case the first `padding` suffices.

Reply:
good catch, this was a quick commit but you're right it doesn't make sense, i'll fix it since there's a weird test error i need to fix before merge anyway 👍

Reply:
Removed the unnecessary property 👍